code/game/objects/items/devices/traitordevices.dm

```diff
@@ -392,6 +399,12 @@ effective or pretty fucking useless.
 	GLOB.active_jammers -= src
 	return ..()
 
+/obj/item/jammer/makeshift
+	name = "makeshift radio jammer"
+	desc = "A jury-rigged device that disrupts nearby radio communication. Its crude construction provides a significantly smaller area of effect compared to its Syndicate counterpart."
+	range = 5
+	disruptor_range = 3
+
```

Comment:
```suggestion

/obj/item/jammer/makeshift/Initialize(mapload)
	. = ..()
	ADD_TRAIT(src, TRAIT_CONTRABAND, INNATE_TRAIT)

```
As a way to keep these as a conspicuous item and to incentivize security to recognize that these are illegal in-game, best to throw on the contraband trait (Which officers can scan in-game to determine they're illegal)